parallel/client.py

```diff
@@ -186,3 +187,16 @@ def submit(self, task_fn, *args):
         future = self.stubs[self._idx].Submit.future(request)
         self._idx = (self._idx + 1) % len(self.stubs)
         return BinaryFutureWrapper(future)
+
+    def verify(self):
```

Comment:
Currently this does does not verify the host that is running the client. This seemed helpful so if you wanted to modify your local 'analysis' code while still using a homogenous set of workers to generate data.

Reply:
This makes sense to me. Homogeneity amongst the workers is the main thing we care about.